tasks.py

```diff
@@ -0,0 +1,60 @@
+import csv
+import time
+from urllib.parse import urlparse
+from robocorp.tasks import task
+from robocorp import browser
+
+
+@task
+def take_website_screenshots():
+    """Take screenshots of all websites found in websites.csv file."""
+    with open("websites.csv", mode="r") as csv_file:
+        csv_reader = csv.DictReader(csv_file)
+        websites = list(csv_reader)
+
+    for website in websites:
+        browser.goto(website["url"])
+        page = browser.page()
+
+        accept_cookies_and_consents(website)
+
+        # some websites have animations on cookies and consents so wait for that to disappear
+        time.sleep(1)
+
+        domain = get_domain_name(website["url"])
+        page.screenshot(path=f"output/{domain}.png", full_page=True)
+
+
+def get_domain_name(url) -> str:
+    parsed_url = urlparse(url)
+    domain_parts = parsed_url.netloc.split(".")
```

Comment:
Nit
```suggestion
    return parsed_url.netloc.replace(".", "_")
```
in case you become afraid of clashes like:
- site.com vs. site.ro
- app.site.com vs. news.site.com
where in each case a `site.png` image will be generated.

Reply:
The alternative here is to leave only the network location as it was in the robot.yaml example

Reply:
That doesn't sound bad, the more explicit the better, just don't forget to replace the dots (`.`) with something like underscores.